.changeset/twenty-queens-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+Added support for templating secrets into actions input, and also added an extra `token` input argument to all publishers to provide a token that would override the `integrations.config`
```

Comment:
should this perhaps also link out to the docs section that you updated in this PR?